sw/airborne/modules/loggers/sdlog_chibios/usbStorage.c

```diff
@@ -205,16 +191,15 @@ static const USBDescriptor* getDescriptor(USBDriver* usbp, uint8_t type, uint8_t
 }
 
 /* Handles global events of the USB driver */
-static void usbEvent(USBDriver* usbp, usbevent_t event)
+static void usbEvent(USBDriver *usbp, usbevent_t event)
 {
   (void) usbp;
 
-  switch (event)
-  {
+  switch (event) {
     case USB_EVENT_CONFIGURED:
-      chSysLockFromIsr();
+      chSysLockFromISR();
```

Comment:
As stated by the module's name, it only works with chibios. There is no arch indep API for mass storage. If it is ever needed, I'll improve this part as well.